crates/abd-clam/src/core/graph/_graph.rs

```diff
@@ -795,8 +795,8 @@ mod tests {
             let selected_clusters = select_clusters(
                 raw_tree.root(),
                 &pretrained_models::get_meta_ml_scorers().first().unwrap().1,
-                i,
-            );
+            )
+            .unwrap();
```

Comment:
Don't use `unwrap` here. Use the `?` operator

Reply:
Since this is a test function that does not return an error you can not use the ? to propagate the error